src/aleph/http/schema.clj

```diff
@@ -0,0 +1,46 @@
+(ns ^:no-doc aleph.http.schema
+  (:require [malli.core :as m]))
+
+(def server-port [:maybe :int])
+(def server-name [:maybe :string])
+(def remote-addr [:maybe :string])
+(def uri [:maybe :string])
+(def query-string [:maybe :string])
+(def scheme [:enum :http :https])
+(def request-method :keyword)
```

Comment:
Note that Netty actually [defines an enum for this](https://github.com/netty/netty/blob/4.1/codec-http/src/main/java/io/netty/handler/codec/http/HttpMethod.java#L88-L101), so allowing arbitrary keywords will lead to an error further down the line. But of course, that property is private so we wold have to redundantly declare it here :grimacing: You call!

Reply:
While an enum is defined, nothing prevents you from creating arbitrary HTTP method. [1]
I would say this enum is just to prevent allocations (maybe?).

[1] : https://github.com/netty/netty/blob/4.1/codec-http/src/main/java/io/netty/handler/codec/http/HttpMethod.java#L117-L118

Reply:
Ah thanks, should have read `valueOf` more closely! Works for me then :+1: